CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+# Changelog
+
+Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and 
+[Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.
+
+
+## 0.1.1 - 2022-07-05
```

Comment:
```suggestion
## 0.2.0 - 2022-07-11
```

Reply:
Once merged, please create a tag and manual release.

Reply:
Sure, will do